Open_Science_New

```diff
@@ -413,7 +398,7 @@
 		    <p><small> “LORIS is a modular and <font color="f27c30">extensible</font> web-based data management system that integrates all aspects of a <font color="f27c30">multi-center</font> study: from <font color="f27c30">heterogeneous data acquisition</font> (imaging, clinical, behavior, genetics) to storage, <font color="f27c30">processing</font> and ultimately <font color="f27c30">dissemination</font>.”</small></p>
 		  </td>
 		  <td>
-		    <img src="images/LORIS_paper.png"> 
+		    <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3262165/"><img src="images/LORIS_paper.png"></a>
```

Comment:
this paper is hosted on our Loris.ca site.  use the URL there (so that people without academic access can always download it) : 
http://loris.ca/images/LORIS-paper-frontiers.pdf

Reply:
thanks, done